ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/documentation/devices/SITE1-LER2.md

```diff
@@ -296,6 +296,7 @@ interface Ethernet8
 
 | Interface | Ethernet Segment Identifier | Multihoming Redundancy Mode | Route Target |
 | --------- | --------------------------- | --------------------------- | ------------ |
+| Port-Channel3 | 0000:0000:0102:0000:0034 | all-active | 01:02:00:00:00:34 |
```

Comment:
is this expected that we did not have it rendered before?

Reply:
ok looks like doc was not doing it properly. that fixes this at the same time!

Basically we do check for `esi` in the documentation template (https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/port-channel-interfaces.j2#L167)  but sadly the interface is not added to the list of `evpn_es_po_interfaces` if it does not have an `evpn_ethernet_segment` section: https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/port-channel-interfaces.j2#L144

so line 144 should be fixed as well to handle "legacy" case in documentation. @ClausHolbechArista should we take this PR as an opportunity to fix it?

Reply:
let's ignore this.